backend/src/services/data_fetcher.py

```
@@ -373,6 +384,8 @@ async def _fetch_token_security(self, token_address: str) -> Optional[Dict]:
                 data = response.json()
                 if data.get('success'):
                     return data.get('data')
+                else:
+                    logger.warning(f"Birdeye Security API returned success=False for {token_address}: {data.get('message')}")
         except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)